### DISCUSSION TITLE
Export pre-populated databases

### DISCUSSION THREAD

**Discussion by marcosvega91** [mswjs/data#204] [Ideas]
We can provide some api to generate handles, entities, models, for example I can create Pokemon API by using

```js

import { createPokemonAPIs } from '@mswjs/data/pokemon'

createPokemonAPIs(mswInstance)

```

The function will create a new db with pokemon entities, populate with some data and attach to msw instance the handlers.
We can add more examples that can be used for POC applications

**Comment by kettanaito** [comment]
Hey, @marcosvega91. I think having a pre-defined set of models is a good idea. Let's tackle this once the main functionality of this library is implemented? I feel like right now we are in the design phase, and things may shift and change. Populating such ready-to-use presets is better on a relatively stable library.

**Comment by marcosvega91** [comment]
Yes yes it was only a way to track this feature
